Sunrise/Cells/CartLineItemCell.swift

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright (c) 2016 Commercetools. All rights reserved.
+//
+
+import UIKit
+
+class CartLineItemCell: UITableViewCell {
+
+    @IBOutlet weak var productNameLabel: UILabel!
+    @IBOutlet weak var productImageView: UIImageView!
+    @IBOutlet weak var priceLabel: UILabel!
+    @IBOutlet weak var quantityLabel: UILabel!
+    @IBOutlet weak var totalPriceLabel: UILabel!
+    @IBOutlet weak var skuLabel: UILabel!
+    @IBOutlet weak var sizeLabel: UILabel!
+
+    override func prepareForReuse() {
+        super.prepareForReuse()
+
+        productImageView.image = nil
```

Comment:
From the docs it says `For performance reasons, you should only reset attributes of the cell that are not related to content, for example, alpha, editing, and selection state.`, thus we should reset the imageView in the controller only.